source/features/latest-tag-button.tsx

```diff
@@ -109,8 +108,8 @@ async function init(): Promise<false | void> {
 		link.append(' ', <span className="css-truncate-target">{latestTag}</span>);
 	}
 
-	link.classList.add('tooltipped', 'tooltipped-ne');
```

Comment:
I moved this into the if statement's since it can be blank

Reply:
This would be a bugfix, not linting anymore. New PR is ok